nevergrad/benchmark/experiments.py

```diff
@@ -143,20 +142,20 @@ def yawidebbob(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
     # Discrete, unordered.
     for nv in [10, 50, 200]:
         for arity in [2, 7]:
-            instrum = ng.p.TransitionChoice(range(arity), repetitions=nv)  # type: ignore
+            instrum = ng.p.TransitionChoice(range(arity), repetitions=nv)
             for discrete_func in [corefuncs.onemax, corefuncs.leadingones, corefuncs.jump]:
                 dfunc = ExperimentFunction(discrete_func, instrum)
                 dfunc._descriptors.update(arity=arity)
                 for optim in optims:
                     for nw in [1, 10]:
                         for budget in [500, 5000]:
                             yield Experiment(dfunc, optim, num_workers=nw, budget=budget, seed=next(seedg))
-    mofuncs: tp.List[PackedFunctions] = []
-
     # The multiobjective case.
+    # TODO the upper bounds are really not well set for this experiment with cigar
```

Comment:
@teytaud I have added this note, because this section is not very interesting: cigar ruins it since it has very high values. I'll let you update/remove as you wish